rosdep/base.yaml

```diff
@@ -2172,6 +2177,7 @@ liburdfdom-headers-dev:
     trusty: [liburdfdom-headers-dev]
     utopic: [liburdfdom-headers-dev]
     vivid: [liburdfdom-headers-dev]
+  gentoo: [media-libs/urdfdom_headers]
```

Comment:
Where these (`console_bridge`, `urdfdom`, and `urdfdom_headers`) contributed by you or someone else into portage? Any idea what versions?